jib-core/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 ### Changed
+- `JibContainerBuilder#addDependencies` is now split into three methods: `addDependencies`, `addSnapshotDependencies`, `addProjectDependneices` ([#1773](https://github.com/GoogleContainerTools/jib/pull/1773))
```

Comment:
nit: add blank line after `### Changed`

Reply:
And fix spelling error in "dependencies"